### ISSUE TITLE
Fichier trop gros pour être téléchargé ?

### ISSUE THREAD

**Issue by Draky50110** [broncowdd/BoZoN#26]
Salut

J'ai envoyé dans le répertoire "uploads" un fichier video.mkv d'environ 350 Mo mais quand je veux le télécharger, ça finit en échec.

Pour tester : http://gilles.wittezaele.fr/bozon/?f=155e19f99548ca

**Comment by broncowdd** [comment]
Je viens de tester sur un webbook 1go ram/1.60GHz et ça tourne impec sans aucun ralentissement... 700 Mo en quelques secondes...
Je vais tester sur mon Rpi pour voir.

**Comment by cborne** [comment]
je passe par une redirection pour ma part un domaine DTDNS ce qui veut 
dire que quand je télécharge chez moi, il considère que je suis pas chez 
moi, je ne sais pas si je suis clair.

**Comment by broncowdd** [comment]
oui, c'est clair... ^^ T'as toujours des configs exotiques: de l'or en barre pour les tests ^^

**Comment by cborne** [comment]
oui mais c'est quand même à terme ce qui va se passer, le gars il 
télécharge pas de chez lui par bozon, il télécharge chez lui mais dehors.

**Comment by broncowdd** [comment]
ben oui ^^
Sinon, autant avoir une clé USB :-D

**Comment by broncowdd** [comment]
Je reteste en wifi au lieu du filaire.

**Comment by Draky50110** [comment]
Z'êtes pas en train de pourrir mon issue là ? :p 
@cborne et @broncowdd vous arrivez à DL le fichier dont j'ai mis le lien ou pas ?

**Comment by broncowdd** [comment]
Hinhinhin DSL ;-) 
Non, en effet, le dl ne commence même pas vraiment...

**Comment by cborne** [comment]
le download tombe en échec direct ce qui était presque mon cas. Cela dit 
si tu regardes les autres issues dans lesquelles on intervient à deux, 
tu remarqueras qu'on prend quand même nos aises.

**Comment by Draky50110** [comment]
Oui j'avais lu les autres issues :p 
@broncowdd tu as pas de compte jabber ? histoire qu'on chat et en plus @cborne qui ralait parce qu'on chattait via email...

**Comment by broncowdd** [comment]
hinhinhin ... pas con ^^ 
J'ai pas reconfiguré pidgin depuis mon réinstall et je sais plus où j'ai fourré mes ID...
http://chat.warriordudimanche.net/chatroom.php?id=5d769c9db59320baf737#cHlq9QKb6Ffy8wPnA5cShbLyu/ODdYkNf/noYqgzCa8=
